### DISCUSSION TITLE
Karrio public roadmap

### DISCUSSION THREAD

**Discussion by danh91** [karrioapi/karrio#241] [Announcements]
Hey everyone,

As promised, we have a public roadmap created with Github projects here https://github.com/orgs/karrioapi/projects/12/views/1
All issues (bugs + features) will be attached there so that anyone can keep track of them and get an idea of what to expect from the monthly releases.

**Comment by manojks148** [comment]
Hi,

What is the default admin login for dashboard ?

**Reply by danh91** [reply]
Hey @manojks148,
The default username password are

admin@example.com | demo
